src/app/views/events/common/evidenceGridIdCell.tpl.html

```diff
@@ -3,7 +3,7 @@
      tooltip-popup-delay="{{ctrl.tooltipPopupDelay}}"
      tooltip-placement="right"
      tooltip-append-to-body="true">
-  <span class="btn-block" ng-class="{ 'statusAccepted': row.entity['status'] === 'accepted', 'statusSubmitted': row.entity['status'] === 'submitted', 'statusRejected': row.entity['status'] === 'rejected' }">
+  <span class="btn-block" ng-class="{ 'statusAccepted': row.entity['status'] === 'accepted', 'statusSubmitted': row.entity['status'] === 'submitted', 'statusRejected': row.entity['status'] === 'rejected' || row.entity['status'] === 'assertion rejected'  }">
```

Comment:
This is not entirely necessary because we updated the assertion status values before the first deploy so no assertion in the live database would have a status of 'assertion rejected'.